rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date, :student, :book
```

Comment:
- [ ] For this section,  Getters and Setters for `@person` instance variables should be present. Kindly refer to this  [resource](https://github.com/microverseinc/curriculumruby/blob/main/oop/articles/oop_principles_examples.md#useful-shortcuts-attr_reader-attr_writer-and-attr_accessor)

Reply:
The link to the resource is not working, can you please reshare the link.

Reply:
well resolved! ✔️

Reply:
Resource link [here](`https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#useful-shortcuts-attr_reader-attr_writer-and-attr_accessor`)